pontoon/batch/utils.py

```diff
@@ -80,7 +82,7 @@ def find_and_replace(translations, find, replace, user):
 
     # No matches found
     if translations.count() == 0:
-        return translations, []
+        return translations, [], []
 
     # Empty translations produced by replace are not allowed for all formats
```

Comment:
I think we remove this block now that we run all the checks (including this one)?

Also, the `return` in line 112 should is a bug (luckily prevented on frontend). It should be `continue`.